charts/logan/templates/ekscp-logs-configmap.yaml

```diff
@@ -0,0 +1,359 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+{{- $kubernetesClusterName := (include "logan.kubernetesClusterName" .) }}
```

Comment:
Shouldn't we render this configmap only if ekscplogs enabled ?

Reply:
Updated